composer/loggers/neptune_logger.py

```diff
@@ -304,4 +330,46 @@ def _validate_image(img: Union[np.ndarray, torch.Tensor], channels_last: bool) -
     if not channels_last:
         img_numpy = np.moveaxis(img_numpy, 0, -1)
 
-    return img_numpy
+    return _validate_image_value_range(img_numpy)
+
+
+def _validate_image_value_range(img: np.ndarray) -> np.ndarray:
+    array_min = img.min()
+    array_max = img.max()
+
+    if (array_min >= 0 and 1 < array_max <= 255) or (array_min >= 0 and array_max <= 1):
+        return img
+
+    from neptune.common.warnings import NeptuneWarning, warn_once
+
+    warn_once(
+        'Image value range is not in the expected range of [0.0, 1.0] or [0, 255]. '
+        'This might be due to the presence of `transforms.Normalize` in the data pipeline. '
+        'Logged images may not display correctly in Neptune.',
+        exception=NeptuneWarning,
+    )
+
+    return _scale_image_to_0_255(img, array_min, array_max)
+
+
+def _scale_image_to_0_255(img: np.ndarray, array_min: Union[int, float], array_max: Union[int, float]) -> np.ndarray:
+    scaled_image = 255 * (img - array_min) / (array_max - array_min)
+    return scaled_image.astype(np.uint8)
+
+
+def _find_oom_callback(callbacks: List['Callback']) -> Optional['OOMObserver']:
+    from composer.callbacks import OOMObserver
+
+    for callback in callbacks:
+        if isinstance(callback, OOMObserver):
+            return callback
+    return None
+
+
+def _warn_about_deprecated_upload_artifacts() -> None:
+    from neptune.common.warnings import NeptuneDeprecationWarning, warn_once
+    warn_once(
+        'The \'upload_artifacts\' parameter is deprecated. '
+        'Please use the \'upload_checkpoints\' parameter instead.',
```

Comment:
```suggestion
        'The \'upload_artifacts\' parameter is deprecated and will be removed in the next version. '
        'Please use the \'upload_checkpoints\' parameter instead.',
```

Reply:
Technically it's already removed, as we don't do literally anything with a passed value

Reply:
Created a PR with the changes I had in mind for supporting `upload_artifacts`: https://github.com/AleksanderWWW/composer/pull/1/files